tests/unit/test_loader_collection.py

```diff
@@ -182,18 +191,24 @@ def readme_artifact_file(request):
 
 @pytest.mark.sha256("e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855")
 def test_check_artifact_file(populated_collection_root, readme_artifact_file):
-    res = chksums_utils.check_artifact_file(populated_collection_root, readme_artifact_file)
+    res = chksums_utils.check_artifact_file(
+        populated_collection_root, readme_artifact_file
+    )
```

Comment:
we want to keep this formatted as-is. `make lint/format/black` should revert these changes automatically.

Reply:
I can re run it but I ran it before I pushed my final commit @awcrosby so i thought it would have caught that

Reply:
```make lint/format/black
reformatted tests/unit/test_loader_collection.py

All done! ✨ 🍰 ✨```